app/src/main/res/layout/fragment_send_confirm.xml

```diff
@@ -88,4 +88,13 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.48" />
+    <ImageView
+        android:layout_width="401dp"
```

Comment:
This PR is awesome!

The only thing that stood out was maybe don't hardcode the size, based on DP but rather use percentages. I can submit an example tomorrow, if it helps but you can make the width relative to the height and the size will probably work better on more screens.